clean.bat

```diff
@@ -0,0 +1,4 @@
+echo y|rmdir /s MyCompany.Seeq.Link.Connector.MyConnector\bin 
+echo y|rmdir /s MyCompany.Seeq.Link.Connector.MyConnector\obj
+echo y|rmdir /s c:\seeq-connector-sdk\csharp-new\Seeq.Link.SDK.Debugging.Agent\bin 
+echo y|rmdir /s c:\seeq-connector-sdk\csharp-new\Seeq.Link.SDK.Debugging.Agent\obj
```

Comment:
We may want to consider whether or not the clean should delete also nuget and/or the packages folder.

Reply:
I see no reason to remove the packages folder. This usually comes from nuget and cannot change.